routes/admin.php

```diff
@@ -91,6 +91,7 @@
 
         // WEB-1963: Browser for nested modules must be implemented manually
         Route::get('digitalPublicationsFoo/{digitalPublication}/articles/browser', [DigitalPublicationArticleController::class, 'getSubbrowserItems'])->name('collection.articles_publications.digitalPublications.articles.subbrowser');
```

Comment:
@web-dev-trev I know this is a change from a different branch, but the route here starts with `digitalPublicationsFoo` which I imagine is some leftover debugging code.

Reply:
Maybe we should give this different prefix/suffix so it doesn't look like a temporary testing route. `digitalPublicationsBrowser/{digitalPublication}/articles/browser`?